babel/messages/catalog.py

```diff
@@ -93,10 +93,10 @@ def __init__(self, id, string=u'', locations=(), flags=(), auto_comments=(),
                        PO file, if any
         :param context: the message context
         """
-        self.id = id #: The message ID
```

Comment:
Hmm, I think `#:` comments should be left as `#:` (and not turned into `# :`), because some tools (as far as I know) use them for some sort of automated documentation generation.